docs/fedora-releases-guide/index.md

```diff
@@ -161,18 +161,23 @@ corresponding to the release that triggered the job. If the version and tag diff
 you can specify the [`upstream_tag_template`](/docs/configuration/#upstream_tag_template)
 configuration option so that Packit can extract the correct version.
 
+The changelog entry generated by default is `- Update to version <version>`.
 If you use [`copy_upstream_release_description: true`](/docs/configuration/#copy_upstream_release_description),
 the changelog entry will use the GitHub/GitLab release description field.
 (Just make sure the formatting is compatible with spec file.
 E.g. use `-` instead of `*` for lists to not create multiple changelog entries.)
 There is also [`sync_changelog`](/docs/configuration/#sync_changelog) configuration option to enable syncing 
 the whole changelog.
-You can also utilize a [custom `changelog-entry` action](/docs/configuration/actions#syncing-the-release).
+You can also utilize a [custom `changelog-entry` action](/docs/configuration/actions#syncing-the-release) if you would like to override the default behaviour.
 
         actions:
           changelog-entry:
             - bash -c 'echo "- New release ${PACKIT_PROJECT_VERSION}"'
 
+Besides the configuration of changelog via Packit, you can also use `%autochangelog` macro to
+generate the changelog entries automatically, see 
+[details](https://docs.pagure.org/Fedora-Infra.rpmautospec/autochangelog.html). In that case, Packit will not touch 
+the changelog at all.
```

Comment:
```suggestion
In case you prefer to use `%autochangelog` macro to generate the changelog entries automatically, see 
[details](https://docs.pagure.org/Fedora-Infra.rpmautospec/autochangelog.html), you can do that and Packit will not touch the changelog at all.
```

Reply:
Ok, then can I suggest this (*besides* just doesn't feel right to me)?

```suggestion
Apart from updating changelog via Packit, you can also use `%autochangelog` macro to
generate the changelog entries automatically, see 
[details](https://docs.pagure.org/Fedora-Infra.rpmautospec/autochangelog.html). In that case, Packit will not touch 
the changelog at all.
```